anaconda.spec.in

```diff
@@ -144,7 +144,7 @@ Requires: python3-coverage >= 4.0-0.12.b3
 Requires: anaconda-tui = %{version}-%{release}
 
 # Make sure we get the en locale one way or another
-Requires: (glibc-langpack-en or glibc-all-langpacks)
+Requires: glibc-langpack-en
```

Comment:
It is, since we want all of them available so the welcome can setlocale to near anything. glibc-all-langpacks is also supposed to be the package chosen during an upgrade from old, non-langpacked glibc, and glibc-all-langpacks is in a few of the desktop environment groups, including @workstation-product.